src/Storage/StorageClient.php

```diff
@@ -4,13 +4,13 @@
 
 class StorageClient extends StorageBucket
 {
-	public function __construct($url, $headers)
+	public function __construct()
```

Comment:
The signature of this class should be:
```
public function __construct($reference_id, $api_key, $bucket_id)
```

Then we should pass in those vars when instantiated. This makes it impossible to use this as a library without the DotEnv library. For testing, we can setup a .env.test that can contain those keys.